### ISSUE TITLE
Change overall size of Parallax Slider

### ISSUE THREAD

**Issue by rostam777** [jlethuau/wp-parallax-content-slider#34]
Hi, I just saw your support notice after I posted my qustion on the plugin page's support forum. 
So here again :-) can you please tell me how I can change the size of your slider?

I am using the wcolor theme and I want the slider to have the full size of the area where it is right now. So the elements would slide in from the right wiewport corner, stop in the middle and slide out left.... but right now there is just a very small slider. I attach two images so you can see what I mean :-)

![parallax_1](https://f.cloud.github.com/assets/5278516/1001182/3fc86e1e-0a63-11e3-8af2-5b269eb51ae1.jpg)

![parallax_2_wcolortheme](https://f.cloud.github.com/assets/5278516/1001181/3fc152b4-0a63-11e3-8e2c-a5e2d5b8177e.jpg)

Ok, for the first step I found out how to delete the "Homepage Title and stuff" from the wcolor theme, which were left to the slider. Also I found where in your css the background picture (waves.gif) was. I deleted this for the moment. But there are still orange top and bottom borders and I don't know where they come from. And still there is the "size-issue". As you see in the picture the width is actually at 100%... but the result is something different.... you have any solution for this?

![parallax_3](https://f.cloud.github.com/assets/5278516/1001183/3fd9a74c-0a63-11e3-88f2-0d267f9e619c.jpg)

Appreciate your help! Thank you in advance :-)

Kind regards from Germany ;)

**Comment by alex80ks** [comment]
Do you want something like this? www.velikisiljegovac.com?

**Comment by rostam777** [comment]
Hi,
thank you for your reply. Not exactly like the example you sent. The
sliding background width should match the whole viewport width, Only the
parallax content images should be in the middle. Right now, when you expand
the background image's width, also the content width expands with it. You
know what I mean? I can try to draw it for you if it's not intelligible.

Cheers
Rostam

2014-02-14 18:04 GMT+01:00 alex80ks notifications@github.com:

**Comment by alex80ks** [comment]
You want slider content to be centered no matter what size viewport is?

**Comment by rostam777** [comment]
Exactly! :-) only the content, but background full 100% width. I've made a
quick concept sketch to better show what I mean.

Do you think this is possible with your plugin?

Cheers

Rostam

Am 15.02.2014 00:05 schrieb "alex80ks" notifications@github.com:

**Comment by alex80ks** [comment]
Sorry but im not the owner of this plugin :) im just a user. 
First you must know your theme files and css and where you want to put it.
For example i want to put him outside content (after navigation and before body or you want to put it in page with other content).

 Also you must know will the background of slider be transparent then will be using your theme background and you wont see background sliding except text,link and image

and all that demand little more serious changesof your theme and maybe little bit of slider depending on position of slider where do you want to be.

**Comment by alex80ks** [comment]
can i see that sketch?

**Comment by rostam777** [comment]
Ah allright, I thought you are the owner :-)

Well unfortunately I am quite new in working with wordpress... and I'm
still learning.... so I'm not that sure this is something I can do these
changes by myself now :(

2014-02-15 1:13 GMT+01:00 alex80ks notifications@github.com:

**Comment by rostam777** [comment]
didn't I send it with the E-Mail before where I mentioned it?

Ok, I'll attach again. Hope you receive it now :-)

2014-02-15 1:14 GMT+01:00 alex80ks notifications@github.com:

**Comment by alex80ks** [comment]
I havent get anything...my email is same as my username just add @ gmail.com

**Comment by rostam777** [comment]
got the email I sent to alex80ks@gmail.com ?

2014-02-15 1:28 GMT+01:00 alex80ks notifications@github.com:
